daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -432,7 +432,7 @@ object Repl {
             println(s"time: ${diff}ms")
             if (!errored) {
               val result = machine.ctrl match {
-                case Speedy.CtrlValue(sv) =>
+                case CtrlValue(sv) =>
```

Comment:
I prefer to leave things like this in actual feature PRs unchanged. If you can't resist your temptation to change it, please put it in a separate PR which does nothing but these style corrections. Having two separate PRs makes reviewing easier.

Reply:
This was unintended.
An accidental hangover from splitting PRs. I'll revert this change back.